101-redis-cache/azuredeploy.json

```diff
@@ -54,6 +54,13 @@
         "description": "A boolean value that indicates whether to allow access via non-SSL ports."
       }
     },
+    "minimumTlsVersion": {
+      "type": "string",
+      "defaultValue": "1.0",
```

Comment:
any reason we can't default this to 1.2?

Reply:
This would be a breaking change for customers already using 1.0.

Reply:
How?  Since this is deploying a new instance (and they can override the defaultValue?)